client/verta/tests/registry/model_version/test_crud.py

```diff
@@ -378,3 +385,37 @@ def test_batch(self, model_version, strs):
             key2: attr2,
             key3: attr3,
         }
+
+
+class TestModelIODescription:
+    def test_input_description(self, client, created_entities):
+        registered_model = client.set_registered_model()
+        created_entities.append(registered_model)
+        model_version = registered_model.get_or_create_version(name="my version")
```

Comment:
I know this might be a bit confusing and existing tests are a bit inconsistent 😭 but we should try using `create_*()` when creating new entities.

```suggestion
        registered_model = client.create_registered_model()
        created_entities.append(registered_model)
        model_version = registered_model.create_version(name="my version")
```